views/lib/help_menu.hbs

```diff
@@ -1,61 +1,65 @@
 <li class="nav-item help-drodown">
-    <div class="dropdown minimal-button">
-        <a class="btn btn-secondary btn-thin dropdown-toggle" title="{{$t "lib.help_menu.label.help"}}" data-toggle="dropdown">
-            <i class="fa fa-question"></i>
-        </a>
-        <div class="dropdown-menu">
-            <ul class="content list-group">
-                <li class="list-group-item list-group-item-action">
-                    <span>
-                        <i class="fa fa-question-circle" aria-hidden="true"></i>
-                        <a href="/help" class="link">{{$t "lib.help_menu.link.helpArea" }}</a>
-                    </span>
-                </li>
+	<div class="dropdown minimal-button">
+			<a class="btn btn-secondary btn-thin dropdown-toggle"
+		   	   href="" title="{{$t "lib.help_menu.label.help"}}" aria-label="{{$t "lib.help_menu.label.help"}}"
+		       data-toggle="dropdown">
+				<i class="fa fa-question" aria-hidden="true"></i>
+			</a>
+		<div class="dropdown-menu">
+			<ul class="content list-group">
```

Comment:
atm I am not able to navigate through the dropdown via arrow keys, just wondering if this is needed..

Reply:
As decided this should be fixed in a separate PR -> https://ticketsystem.schul-cloud.org/browse/SC-5858 
I will mark this comment as resolved then.